CONFIG.md

```diff
@@ -13,6 +13,7 @@ stick to [scrapyd's configuration](https://scrapyd.readthedocs.io/en/latest/conf
 * `username`     - Set this and `password` to enable basic authentication ([➽](https://scrapyd.readthedocs.io/en/latest/config.html#username))
 * `password`     - Set this and `username` to enable basic authentication ([➽](https://scrapyd.readthedocs.io/en/latest/config.html#password))
 * `log_level`    - Log level, defaults to `INFO`
+* `logs_dir`     - a directory to store log files collected on k8s cluster (implemented only for Kubernetes), if not provided there is a default value `/tmp/scrapyd_k8s_logs`. When configuring, keep in mind that in the Dockerfile the `USER` is set to `nobody` so not all directories are writable, but if you make a child directory under `/tmp` you won't encounter permission problems.
```

Comment:
Yes, correct, I will move the details to the specific section and move the parameter itself, it does belong only to the joblogs because we do not really use it anywhere else.